src/test/groovy/com/florianmski/suncalc/SunCalcSpec.groovy

```diff
@@ -61,16 +63,15 @@ class SunCalcSpec extends spock.lang.Specification {
 
     }
 
-    @Ignore("original Moon test code ported, but not yet fully tested")
+    @Ignore("original Moon Java test code ported, but not yet fully tested")
```

Comment:
Haha so this was a port from the original JUnit test, and looking at https://github.com/mourner/suncalc/blob/master/test.js it doesn't seem to match any of the JavaScript tests... Is that confusing, though?